recipes/clyent/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.2.1" %}
+
+package:
+  name: clyent
+  version: {{ version }}
+
+source:
+  fn: clyent-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/clyent/clyent-{{ version }}.tar.gz
+  md5: a8a953a636cbd75021c3983f7ca60b5f
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - clyent
+
+about:
+  home: https://github.com/Anaconda-Server/clyent
+  license: BSD
```

Comment:
Sounds good. Let's see what they say. Added a ref to the issue below.

xref: https://github.com/Anaconda-Platform/clyent/issues/11